### ISSUE TITLE
Error with ruby 2.4 and the --livereload flag on Windows

### ISSUE THREAD

**Issue by demedos** [jekyll/jekyll#7221]
- [x] I believe this to be a bug, not a question about using Jekyll.
- [x] I updated to the latest Jekyll (or) if on GitHub Pages to the latest `github-pages`

---

- [x] I am on (or have tested on) ***Windows*** 10+

---

- [x] It was another bug.

## My Reproduction Steps

ENV: Ruby 2.4 + jekyll 3.8.3

```
jekyll new mysite
cd mysite
jekyll s --livereload
```

Throws error: `Unable to load the EventMachine C extension; To use the pure-ruby reactor, require 'em/pure_ruby`.

After fixing with the solution proposed [here](https://github.com/tabler/tabler/issues/155) (add `require 'em/pure_ruby'` to `eventmachine.rb`) it worked, but when editing a SCSS file it threw a `Bad File Descriptor` error and turned down the server. 
As ashmaroli says in [this post](http://talk.jekyllrb.com/t/bad-file-descriptor-when-using-livereload/1627), the reason in that `EventMachine is not yet compatible with Ruby 2.4+ on Windows`, so I had to install Ruby 2.3.3 and the DevKit manually + update my Ruby executable references in the Environment variables and re-run `gem install jekyll bundler` to make it work.
Hope this helps 

## The Output I Wanted

just live reload scss files

**Comment by ashmaroli** [comment]
@demedos At first glance, this does not seem like a "bug report", but more like "helpful solution / hint / workaround".
If that is indeed the case, I suggest that you open a PR for updating our official docs by cloning this repository locally and adding a note under the following `document` and `section`:
```
docs/_docs/troubleshooting.md#problems-running-jekyll
```
Thank you very much.

---
P.S. If you still have **Ruby 2.4** and the gem `eventmachine` installed under that version, can you verify that a non-empty `lib/2.4/` directory exists in the gem's installation directory (`bundle show eventmachine` should help). *If not, I recommend that you open a new issue-ticket at [eventmachine's repo](https://github.com/eventmachine/eventmachine).*

**Comment by demedos** [comment]
@ashmaroli Indeed this is a workaround to a pretty edge case, I just thought that it would help to have it posted somewhere. Unfortunately I do not have time to update the offical documents, but thank you for your suggestions.

**Comment by ashmaroli** [comment]
Thank you for reporting this anyways @demedos 
We'll wait till `eventmachine-1.3.0` is shipped to be sure that EventMachine on Ruby 2.4+ is faulty..
